saml2/saml2-service-provider/src/main/java/org/springframework/security/saml2/provider/service/authentication/AbstractSaml2AuthenticationRequest.java

```diff
@@ -89,6 +93,14 @@ public String getAuthenticationRequestUri() {
 		return this.authenticationRequestUri;
 	}
 
+	/**
+	 * Returns the id of this AuthNRequest.
+	 * @return the id of this request
+	 */
```

Comment:
Please add `@since 5.7` here and to any other public methods, fields, etc. that you are introducing.